templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>Login</h2>
+  {% if form.errors %}
+    <p style="font-weight: bold; color: red;">
+      Invalid credentials. Try again!
+    </p>
+  {% endif %}
+  <form action="{% url 'login' %}" method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <input type="submit" value="Submit">
+    <input type="hidden" name="next" value="{{ next }}">

```

Comment:
The hidden input field for `next` is used to redirect users after successful login. Ensure that the `next` parameter is correctly passed to the template context when needed.